src/main/java/seedu/address/model/customer/Car.java

```diff
@@ -4,54 +4,76 @@
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's preferred car in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidCar(String)}
+ * Represents a car in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidCar(String, String)}
  */
 public class Car {
 
-    public static final String MESSAGE_CONSTRAINTS = "Car can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Car Brand + Car Type can take any values, it should not be blank";
 
     /*
      * The first character of the preferred car must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
     public static final String VALIDATION_REGEX = "[^\\s].*";
 
-    public final String carId;
+    public final String carBrand;
+    public final String carType;
 
     /**
      * Constructs a {@code Car}.
      *
-     * @param car A valid car.
+     * @param carBrand A valid carBrand.
+     * @param carType A valid carType.
      */
-    public Car(String car) {
-        requireNonNull(car);
-        checkArgument(isValidCar(car), MESSAGE_CONSTRAINTS);
-        carId = car;
+    public Car(String carBrand, String carType) {
+        requireNonNull(carBrand);
+        requireNonNull(carType);
+        checkArgument(isValidCar(carBrand, carType), MESSAGE_CONSTRAINTS);
+
+        this.carBrand = carBrand;
+        this.carType = carType;
+    }
+
+    /**
+     * Returns true if a given string is a valid car.
+     */
+    public static boolean isValidCar(String test1, String test2) {
```

Comment:
Method variables `test1` and `test2` should be descriptive, like `testBrand` and `testType`

Reply:
agreed, changed to Car(String carBrand, String carType)